editor/editor_config.cpp

```diff
@@ -120,9 +120,22 @@ EditorConfig::EditorConfig(string const & fileName)
   Reload();
 }
 
-bool EditorConfig::GetTypeDescription(vector<string> const & classificatorTypes,
+bool EditorConfig::GetTypeDescription(vector<string> && classificatorTypes,
```

Comment:
Макс, там код типа такого GetTypeDescription(CreateVector())
Разве мув нужен?

> On Apr 1, 2016, at 17:03, mpimenov <notifications@github.com> wrote:
> 
> In editor/editor_config.cpp <https://github.com/mapsme/omim/pull/2670#discussion_r58209572>:
> 
> > @@ -120,9 +120,22 @@ EditorConfig::EditorConfig(string const & fileName)
> >    Reload();
> >  }
> >  
> > -bool EditorConfig::GetTypeDescription(vector<string> const & classificatorTypes,
> > +bool EditorConfig::GetTypeDescription(vector<string> && classificatorTypes,
> move по месту использования?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/2670/files/af10e69d522742081c4007f6da63a022705f438b#r58209572>